### DISCUSSION TITLE
Synched event shows wrong start date

### DISCUSSION THREAD

**Discussion by ziomau10** [kewisch/gdata-provider#507] [Support] [answered]
I have an event put on my calendar by someone else enabled to access my calendar on google.
The event is on 6th of october 2022 from 17:00 to 18:00. The event was (probably) generated on the 3rd of October 2022.
In the calendar shown as starting from the 3rd 17:00 to the 6th 18:00 spanning three days.
On google calendar, as well as on other synchronized calendars (aCalendar and google on mobile), the event is shown correctly.
What should I provide to you for debugging?
Is there some log that can be inspected, or sent?
Thanks for any help.
Regards.
![Clipboard-1](https://user-images.githubusercontent.com/1978719/194827788-bc1c5e5f-1559-4552-a662-fdcf12bcc0ef.jpg)
![Untitled](https://user-images.githubusercontent.com/1978719/194828408-62160cfb-a7f1-40de-9d5b-543f1f2e6e3a.jpg)
![Untitled2](https://user-images.githubusercontent.com/1978719/194828963-3e3d6c44-8eea-4c15-a1b3-1e3ad529adeb.jpg)

**Comment by xjonx1** [comment, marked as answer]
There was an issue with the add-on and is now fixed by the developer. Details here: https://github.com/kewisch/gdata-provider/discussions/519#discussioncomment-4147395
